tests/python_tests/helpers/test_config.py

```diff
@@ -147,6 +150,12 @@ def generate_build_header(
         if pack_addrs is not None:
             header_content.append(f"#define PACK_ADDRS {pack_addrs}")
 
+    if perf_run_type := test_config.get("perf_run_type"):
```

Comment:
```suggestion
    if perf_run_type : test_config.get("perf_run_type"):
```
Nitpick: is the "=" suppose to be there?

Reply:
Yes, it's called [walrus operator](https://www.datacamp.com/tutorial/python-walrus-operator?utm_source=google&utm_medium=paid_search&utm_campaignid=19589720821&utm_adgroupid=157156375191&utm_device=c&utm_keyword=&utm_matchtype=&utm_network=g&utm_adpostion=&utm_creative=733936221335&utm_targetid=dsa-2218886984100&utm_loc_interest_ms=&utm_loc_physical_ms=9198461&utm_content=ps-other~emea-en~dsa~tofu~tutorial-python&accountid=9624585688&utm_campaign=230119_1-ps-other~dsa~tofu_2-b2c_3-emea_4-prc_5-na_6-na_7-le_8-pdsh-go_9-nb-e_10-na_11-na&gad_source=1&gad_campaignid=19589720821) :D